Makefile

```diff
@@ -8,7 +8,9 @@ export CGO_ENABLED=0
 GOCMD = go
 GOBUILD = $(GOCMD) build
 
-PROTOC = protoc --gofast_out=plugins=grpc:$(GOPATH)/src/ --proto_path=$(GOPATH)/src/ $(GOPATH)/src/github.com/TheThingsNetwork/ttn
+#PROTOC = protoc --gofast_out=plugins=grpc:$(GOPATH)/src/ --proto_path=$(GOPATH)/src/ $(GOPATH)/src/github.com/TheThingsNetwork/ttn
+PROTOC = protoc --gofast_out=plugins=grpc:$(GOPATH)/src/ --proto_path=$(GOPATH)/src/:$(GOPATH)/src/github.com/gogo/protobuf/protobuf $(GOPATH)/src/github.com/TheThingsNetwork/ttn
```

Comment:
Please do not include this in this PR. This is either a local development environment issue or a general issue not related to the encoder